server_src/player.js

```diff
@@ -876,21 +779,32 @@ class Player {
         }
     }
 
-    shootMissile () {
+    shootMissile2 (aWeapon) {
         const r = Math.random();
         const bAngle = this.angle;
-        const missile = new Missile(this, r, this.weapons[this.equipped], bAngle);
+        const missile = new Missile(this, r, aWeapon, bAngle);
         missiles[this.sy][this.sx][r] = missile;
         sendAllSector(`sound`, { file: `missile`, x: this.x, y: this.y }, this.sx, this.sy);
     }
 
+    shootMissile () {
+        this.shootMissile2(this.weapons[this.equipped]);
+    }
+
     shootOrb () {
         const r = Math.random();
         const orb = new Orb(this, r, this.weapons[this.equipped]);
         orbs[this.sy][this.sx][r] = orb;
         sendAllSector(`sound`, { file: `beam`, x: this.x, y: this.y }, this.sx, this.sy);
     }
 
+    shootMine2 (aWeapon) {
+        const r = Math.random();
+        const mine = new Mine(this, r, aWeapon);
+        mines[this.sy][this.sx][r] = mine;
+        sendAllSector(`mine`, { x: this.x, y: this.y }, this.sx, this.sy);
+    }
+
```

Comment:
The same as with shootMissile, when I tried to name them both the same way, the npm crashed saying compatibility issues

Reply:
I don't really get why it crashes if I call them both the same way but with different arguments

Reply:
Well, I more or so meant just rename the function to be more clear on what it's purpose actually is, rather than just appending a `2` to it is what I'm getting at.

Reply:
Renamed